### PR TITLE
Feature/jls 6210515296/new nci2i page

### PR DESCRIPTION
**Pull request by ecstech-admin**
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Bug Fix
- [ ] Hot Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Develop newly designed NCI2I page

## Related Tickets & Documents
- Monday Task #6210515296
- https://blackberg-group.monday.com/boards/1931719392/pulses/6210515296

## QA Instructions, Screenshots, Recordings
New page: https://dev-innovation.bgsandbox.com/ecosystem/views/national-centers/
Compare developed page to Figma design
Confirm carousel, map and page, in general functions correctly at different break points

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Checked with [Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/home/accelerating-hci.html
 - _includes/home/featured-successes.html
 - _includes/home/masthead.html
 - _includes/home/national-centers-map.html
 - _includes/home/national-centers.html
 - _includes/home/news-and-events.html
 - _includes/home/our-mission.html
 - _site/assets/css/map-svg/mapsvg.css
 - _site/assets/css/map-svg/nanoscroller.css
 - _site/assets/img/home/5g-tower.png
 - _site/assets/img/home/accelerating-innovation.png
 - _site/assets/img/home/drhci-logo.png
 - _site/assets/img/home/ideas-logo.png
 - _site/assets/img/home/ncchi-logo.png
 - _site/assets/img/home/nci2i-1.jpg
 - _site/assets/img/home/nci2i-10.jpg
 - _site/assets/img/home/nci2i-2.jpg
 - _site/assets/img/home/nci2i-3.jpg
 - _site/assets/img/home/nci2i-4.jpg
 - _site/assets/img/home/nci2i-5.jpg
 - _site/assets/img/home/nci2i-6.jpg
 - _site/assets/img/home/nci2i-7.jpg
 - _site/assets/img/home/nci2i-8.jpg
 - _site/assets/img/home/nci2i-9.jpg
 - _site/assets/img/home/nci2i-masthead.png
 - _site/assets/img/home/necie-logo.png
 - _site/assets/img/home/news/bring-people-ideas-together.jpg
 - _site/assets/img/home/news/capstone-project.jpg
 - _site/assets/img/home/news/disease-prevention-tools.jpg
 - _site/assets/img/home/news/face-of-innovation.jpg
 - _site/assets/img/home/news/first-innovation-center.jpg
 - _site/assets/img/home/news/jesse-award-osborne.jpg
 - _site/assets/img/home/news/legacy-partners.jpg
 - _site/assets/img/home/news/ncchi.jpg
 - _site/assets/img/home/news/personalized-healthcare.png
 - _site/assets/img/home/news/predictive-analytics.png
 - _site/assets/img/home/news/project-convergence.png
 - _site/assets/img/home/news/rideshare-for-veteran-employment.jpg
 - _site/assets/img/home/news/robert-jesse-award.jpg
 - _site/assets/img/home/news/smart-sock.jpg
 - _site/assets/img/home/news/va-health-incubator.png
 - _site/assets/img/home/news/veteran-infertility.jpg
 - _site/assets/img/home/news/veteran-rideshare.jpg
 - _site/assets/img/home/news/vulnerable-veterans-rideshare.jpg
 - _site/assets/img/home/news/xprize-pandemic-alliance.jpg
 - _site/assets/img/home/va-ventures-logo.png
 - _site/assets/img/home/vha-nci2i-logo.svg
 - _site/assets/img/map-svg/default-pin.svg
 - _site/assets/img/map-svg/usa.svg
 - _site/assets/js/mustache-fellows-render.js
 - _site/assets/js/mustache-nci2i-news-render.js
 - _site/assets/js/nci2i-home-news.js
 - _site/assets/js/vendors/lazy/jquery.lazy.min.js
 - _site/assets/js/vendors/map-svg/jquery.mousewheel.min.js
 - _site/assets/js/vendors/map-svg/jquery.nanoscroller.min.js
 - _site/assets/js/vendors/map-svg/mapsvg.functions.js
 - _site/assets/js/vendors/map-svg/mapsvg.min.js
 - _site/assets/sass/vha/pages/home/accelerating-hci/_index.scss
 - _site/assets/sass/vha/pages/home/featured-successes/_index.scss
 - _site/assets/sass/vha/pages/home/masthead/_index.scss
 - _site/assets/sass/vha/pages/home/national-centers-map/_index.scss
 - _site/assets/sass/vha/pages/home/national-centers/_index.scss
 - _site/assets/sass/vha/pages/home/news-and-events/_index.scss
 - _site/assets/sass/vha/pages/home/our-mission/_index.scss
 - assets/css/map-svg/mapsvg.css
 - assets/css/map-svg/nanoscroller.css
 - assets/img/home/5g-tower.png
 - assets/img/home/accelerating-innovation.png
 - assets/img/home/drhci-logo.png
 - assets/img/home/ideas-logo.png
 - assets/img/home/ncchi-logo.png
 - assets/img/home/nci2i-1.jpg
 - assets/img/home/nci2i-10.jpg
 - assets/img/home/nci2i-2.jpg
 - assets/img/home/nci2i-3.jpg
 - assets/img/home/nci2i-4.jpg
 - assets/img/home/nci2i-5.jpg
 - assets/img/home/nci2i-6.jpg
 - assets/img/home/nci2i-7.jpg
 - assets/img/home/nci2i-8.jpg
 - assets/img/home/nci2i-9.jpg
 - assets/img/home/nci2i-masthead.png
 - assets/img/home/necie-logo.png
 - assets/img/home/news/bring-people-ideas-together.jpg
 - assets/img/home/news/capstone-project.jpg
 - assets/img/home/news/disease-prevention-tools.jpg
 - assets/img/home/news/face-of-innovation.jpg
 - assets/img/home/news/first-innovation-center.jpg
 - assets/img/home/news/jesse-award-osborne.jpg
 - assets/img/home/news/legacy-partners.jpg
 - assets/img/home/news/ncchi.jpg
 - assets/img/home/news/personalized-healthcare.png
 - assets/img/home/news/predictive-analytics.png
 - assets/img/home/news/project-convergence.png
 - assets/img/home/news/rideshare-for-veteran-employment.jpg
 - assets/img/home/news/robert-jesse-award.jpg
 - assets/img/home/news/smart-sock.jpg
 - assets/img/home/news/va-health-incubator.png
 - assets/img/home/news/veteran-infertility.jpg
 - assets/img/home/news/veteran-rideshare.jpg
 - assets/img/home/news/vulnerable-veterans-rideshare.jpg
 - assets/img/home/news/xprize-pandemic-alliance.jpg
 - assets/img/home/va-ventures-logo.png
 - assets/img/home/vha-nci2i-logo.svg
 - assets/img/map-svg/default-pin.svg
 - assets/img/map-svg/usa.svg
 - assets/js/mustache-fellows-render.js
 - assets/js/nci2i-home-news.js
 - assets/js/vendors/lazy/jquery.lazy.min.js
 - assets/js/vendors/map-svg/jquery.mousewheel.min.js
 - assets/js/vendors/map-svg/jquery.nanoscroller.min.js
 - assets/js/vendors/map-svg/mapsvg.functions.js
 - assets/js/vendors/map-svg/mapsvg.min.js
 - assets/sass/vha/pages/home/accelerating-hci/_index.scss
 - assets/sass/vha/pages/home/featured-successes/_index.scss
 - assets/sass/vha/pages/home/masthead/_index.scss
 - assets/sass/vha/pages/home/national-centers-map/_index.scss
 - assets/sass/vha/pages/home/national-centers/_index.scss
 - assets/sass/vha/pages/home/news-and-events/_index.scss
 - assets/sass/vha/pages/home/our-mission/_index.scss
 - _includes/dark-header-sub.html
 - _includes/dark-header.html
 - _includes/header-sub.html
 - _includes/header.html
 - _site/assets/css/styles.css
 - _site/assets/js/functions.js
 - _site/assets/sass/vha/pages/home/_index.scss
 - _site/drhci/index.html
 - _site/ideas/index.html
 - _site/index.html
 - _site/nc4ti/index.html
 - _site/ncchi/index.html
 - _site/ncchi/our-team/index.html
 - _site/ncchi/publications/index.html
 - _site/va-ventures/index.html
 - assets/css/styles.css
 - assets/js/functions.js
 - assets/sass/vha/pages/home/_index.scss
 - index.html

### CODE REVIEW

**Comment by RyanCrux** [comment]
Couple things:

-  Arrow button on news stories is missing on mobile:
Screenshot 2024-05-31 at 10 46 19 AM

- There is a small paragraph of text (only on mobile) that is missing under the news header
Screenshot 2024-05-31 at 10 45 56 AM

-button styling under photo images doesn't match figma
Screenshot 2024-05-31 at 10 43 06 AM

- This is probably because the locations may have been updated but the location copy info doesn't match source
Screenshot 2024-05-31 at 10 43 33 AM

- Arrow distance moves with screen size and eventually overlaps  
Screenshot 2024-05-31 at 10 41 13 AM

- too much space between sections here
Screenshot 2024-05-31 at 10 44 30 AM

Overall looking really good! These pages are going to bring such a nice refresh!

**Comment by ecstech-admin** [comment]
Hero carousel:
button styles corrected
rm extra padding between carousel and Our Mission

Map:
location edit: NECIE in Manchester NH (from Monday)
(note: static locations mirror popup style with name bold and other info not bold)

News:
Copy from Figma mobile is a mistake: held over from OAM design. 
Fixed styles so nav buttons are available at all breaks.

**Comment by RyanCrux** [comment]
carousel photo placeholders are smaller than figma:
Screenshot 2024-06-03 at 5 29 39 PM

Map moves and cuts off when selecting location:
Screenshot 2024-06-03 at 5 31 16 PM

forward/back buttons look better in many places but still overlap in one:
Screenshot 2024-06-03 at 5 32 14 PM

**Review by RyanCrux — APPROVED** [review]

File: _includes/home/masthead.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1446 724" width="1446" height="723" fill="none" preserveAspectRatio="xMidYMax meet" role="presentation"><path fill="url(#a)" d="M625.945 722.5H0L681.77 0h763.61L763.609 722.5H625.945Z" style="mix-blend-mode:multiply"/><defs><linearGradient id="a" x1="254.9" x2="-1216.89" y1="-3529.43" y2="540.104" gradientUnits="userSpaceOnUse"><stop stop-color="#C9C9C9"/><stop offset=".04" stop-color="#CCC"/><stop offset=".42" stop-color="#E8E8E8"/><stop offset=".75" stop-color="#F9F9F9"/><stop offset="1" stop-color="#fff"/></linearGradient></defs></svg>
+            </div>                
+        </div>
+    </div>
+    <div class="masthead__border">
+        <div class="block block-color-1"></div>
+        <div class="block block-color-2"></div>
+    </div>     
+</section>
+<!-- AWARDS - START
```
Comment (will-bbg) [inline _includes/home/masthead.html]: Remove commented code. This should not make it to production.

File: _includes/home/masthead.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                </div>
+                <div class="tablet:grid-col-6 mobile:padding-top-6 tablet:grid-offset-1 desktop:padding-top-0">
+                    <div class="masthead__img">
+                        <div class="masthead__img-container">
+                            <picture>
+                                <source media="(max-width:639px)" srcset="assets/img/drhci/bubble-lines-sm.svg">
+                                <source media="(min-width:640px)" srcset="assets/img/drhci/bubble-lines.svg">
+                                <img class="position-absolute circle-lines" src="assets/img/drhci/bubble-lines.svg" alt="Circle lines background">
+                            </picture>
+                            <!-- <img src="assets/img/home/nci2i-1.jpg" class="img-fluid" alt="NCI2I group photo" /> -->
```
Comment (will-bbg) [inline _includes/home/masthead.html]: Remove commented code. This should not make it to production.

File: _includes/home/our-mission.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<!-- NCI2I Mission - START -->
+<!-- <section id="our-mission">
```
Comment (will-bbg) [inline _includes/home/our-mission.html]: Remove commented code. This should not make it to production.

**Review by will-bbg — CHANGES_REQUESTED** [review]
### Masthead
- Masthead has an "h2". Make it an h1".
- On tablet, prev/next buttons appear out of nowhere. Please double check.
Screenshot 2024-06-04 at 10 01 44 AM

### Our Mission
- Reduce Our Mission section padding. Mockup shows 80px.
Screenshot 2024-06-04 at 9 47 37 AM
- Our Mission should be an "h2" not an "h1"

### Logo Section
- Make logos link to websites
- Add alt tags to logo images
- Logo titles do not match the size in mockup
- Double check padding below logo titles for tablet and mobile. Currently does not match mockup

### News and Events
- Title should be an "h2" not an "h1"
- Let's remove the Dr. Osborne article. He is not longer with the program and was asked to be remove from NCCHI as well
- On mobile, buttons are too close to each other. Please see mockup.

**Comment by ecstech-admin** [comment]
Made masthead title h1 (with h2 style) ((**This will need to be done on other pages (IDEAS, DRHCI, VA Ventures)**)
Added breaks for carousel arrow buttons and adjusted positioning as needed

Adjusted "Our Mission" padding to match Figma and made title h2 (with h1 style) ((**This will need to be done on other pages (IDEAS, DRHCI, VA Ventures**))

I added link + alt tags to logo images
Adjusted logo title sizes to match Figma size and padding

Made "News and Events" title h2
Removed 4th news article (Dr. Osborne)
Added breaks and adjusted positioning for carousel buttons

**Review by will-bbg — APPROVED** [review]

**Pull request merged**